apps/routing_settings.go

```diff
@@ -48,3 +49,25 @@ func StickySession(appName string, enable bool) error {
 	io.Statusf("Sticky session has been %sd on %s\n", action, appName)
 	return nil
 }
+
+func RouterLogs(appName string, enable bool) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+
+	_, err = c.AppsRouterLogs(appName, enable)
+	if err != nil {
+		return errgo.Notef(err, "fail to configure router-logs feature")
+	}
+
+	var action string
+	if enable {
+		action = "enable"
+	} else {
+		action = "disable"
+	}
+
+	io.Statusf("Router logs has been %sd on %s\n", action, appName)
```

Comment:
Isn't it how we are supposed to conjugate it? 🤔 
```suggestion
	io.Statusf("Router logs have been %sd on %s\n", action, appName)
```

Reply:
Copied it from force-https :rofl:

Reply:
Found this : 
`If the subject of the sentence is He – She – It or a singular noun (car, bird, child) we use ‘has been‘.
If the subject of a sentence is I – You – We – They or a plural noun (cars, birds, children) we use ‘have been‘.`
So `has been` would be the answer.

Reply:
Isn't it the "logs" that are enabled? Hence "have been"?